guides/v2.1/howdoi/patch_from_github.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+group: howdoi
+title: Create a GitHub patch for a Composer installation
+version: 2.2
+github_link: howdoi/patch_from_github.md
+functional_areas:
+  - Install
+  - System
+  - Setup
+---
+
+Sometimes it takes a while for us to include a bug fix made on GitHub in a Magento 2 Composer release. In the meantime, you can create a patch from GitHub and use the [`cweagans/composer-patches`](https://github.com/cweagans/composer-patches/){:target="\_blank"} plugin to apply it to your Composer-based Magento 2 installation.
+
+<div class="bs-callout bs-callout-warning" markdown="1">
+Always perform comprehensive testing before deploying any unreleased patch.
+</div>
+
+## Create a patch
+
+To create a patch file:
+
+1. Create a `patches/composer` directory in your local project.
+
+1. Identify the GitHub commit or pull request to use for the patch. For example:
+
+   [`2d31571f1bacd11aa2ec795180abf682e0e9aede`](https://github.com/magento/magento2/commit/2d31571f1bacd11aa2ec795180abf682e0e9aede){:target="\_blank"}
+
+1. Append `.patch` or `.diff` to the commit URL. Use `.diff` for a smaller file size. For example:
+
+   [https://github.com/magento/magento2/commit/2d31571f1bacd11aa2ec795180abf682e0e9aede.patch](https://github.com/magento/magento2/commit/2d31571f1bacd11aa2ec795180abf682e0e9aede.patch){:target="\_blank"}
+
+1. Save the page as a file in the `patches/composer` directory.
+
+1. Edit the file and change all paths so that they are relative to the `vendor/<VENDOR>/<PACKAGE>` directory. For example, replace all instances of `app/code/Magento/Payment/` with `vendor/magento/module-payment/`.
```

Comment:
This is not correct, it should say something like: "strip all instances of `app/code/<VENDOR>/<PACKAGE>/` from the path, so you end up with a relative path to the `vendor/<VENDOR>/<PACKAGE>/` directory." So the replacement part is incorrect, that first part of the path should be stripped of and not replaced.
As an addition it might be a good idea to use an example from the patch mentioned above, where you would replace all paths `app/code/Magento/Payment/view/frontend/web/js/view/payment/iframe.js` with `view/frontend/web/js/view/payment/iframe.js`